zulip_bots/zulip_bots/bots/monkeytestit/doc.md

```diff
@@ -0,0 +1,57 @@
+# monkeytest.it bot
+
+## Quick overview
+
+This bot provides a quick way to check a site from a chat.
+Using user's monkeytestit's API  key, user can check a website
+for certain failures that user wanted to see.
+
+## Setting up API key
+
+1. Get your monkeytest.it API key, located in your
+   [dashboard](https://monkeytest.it/dashboard).
+
+2. Open `monkeytestit.conf` file. Create one if somehow it
+   doesn't exist yet.
+
+3. Inside `monkeytestit.conf` is this:
+   ```
+   [monkeytestit]
+   api_key = <api key here>
+   ```
+
+4. Put your API key, replacing `<api key here>`.
+
+5. Save the configuration file.
```

Comment:
You'll want to describe how to launch the program, including the options the options to specify the two config files.

Reply:
Added a section "Running the bot"